README.md

```diff
@@ -59,6 +59,25 @@ Read [How the topic taxonomy works](https://docs.publishing.service.gov.uk/manua
 Content Tagger has a "Projects" function for tagging content in bulk
 to the Topic Taxonomy. It also supports tagging individual pages.
 
+### Links
+
+Content Tagger allows you to define your own concepts via 'links', which
+help to denote relationships between different pieces of content.
+
+For example, there is a ['facet group' associated with the business finder](https://content-tagger.integration.publishing.service.gov.uk/facet_groups).
+The facet group has a number of possible facets (e.g. "Sector / Business Area"),
+each of which have a number of possible facet values (e.g. "Accommodation"),
+which is a [manually curated list in YAML](https://github.com/alphagov/content-tagger/blob/master/lib/data/find-eu-exit-guidance-business.yml).
+
+Content designers follow the [How to tag content with facet values](https://docs.publishing.service.gov.uk/manual/tag-content-with-facet-values.html)
+instructions to tag a particular piece of content (e.g.
+"/government/publications/workplace-rights-if-theres-no-brexit-deal") with a
+particular facet value (e.g. "Accommodation").
+
+They can choose whether or not to trigger an email alert for the tagging by
```

Comment:
Not sure if this is needed as it's already covered in the "How to tag content with facet values" docs.